src/api/signature.js

```diff
@@ -1,79 +1,57 @@
-/* eslint-disable consistent-return */
```

Comment:
This file have too much lint alert, only delete them if there is no alert in this file, thanks.